samples/snippets/src/main/java/com/example/analytics/RunReportWithOrderingSample.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.analytics;
+
+/* Google Analytics Data API sample application demonstrating the ordering of report rows.
+
+See
+https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/runReport#body.request_body.FIELDS.order_bys
+for more information.
+
+Before you start the application, please review the comments starting with
+"TODO(developer)" and update the code to use correct values.
+
+To run this sample using Maven:
+  cd java-analytics-data/samples/snippets
+  mvn compile
+  mvn exec:java -Dexec.mainClass="com.example.analytics.RunReportWithOrderingSample"
+ */
+
+// [START analyticsdata_run_report_with_ordering]
+
+import com.google.analytics.data.v1beta.BetaAnalyticsDataClient;
+import com.google.analytics.data.v1beta.DateRange;
+import com.google.analytics.data.v1beta.Dimension;
+import com.google.analytics.data.v1beta.Metric;
+import com.google.analytics.data.v1beta.OrderBy;
+import com.google.analytics.data.v1beta.RunReportRequest;
+import com.google.analytics.data.v1beta.RunReportResponse;
+
+public class RunReportWithOrderingSample {
+
+  public static void main(String... args) throws Exception {
+    /**
+     * TODO(developer): Replace this variable with your Google Analytics 4 property ID before
+     * running the sample.
+     */
+    String propertyId = "YOUR-GA4-PROPERTY-ID";
+    sampleRunReportWithOrdering(propertyId);
+  }
+
+  // Runs a report of active users grouped by three dimensions, ordered by the total revenue in
+  // descending order.
+  static void sampleRunReportWithOrdering(String propertyId) throws Exception {
+    // Using a default constructor instructs the client to use the credentials
+    // specified in GOOGLE_APPLICATION_CREDENTIALS environment variable.
+    try (BetaAnalyticsDataClient analyticsData = BetaAnalyticsDataClient.create()) {
```

Comment:
For client initialization, please make sure to leave a comment clarifying how to handle multiple requests and close clients when complete: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md#client-initialization

Reply:
@kurtisvg I am not sure if I agree with this, could you please clarify? I agree with maybe adding something like "Initializes the client that will be used to send requests", but not about closing clients when complete. I looked through some of the other samples in the repository you linked, and even though they contain the comment

```
    // Initialize client that will be used to send requests. This client only needs to be created
    // once, and can be reused for multiple requests. After completing all of your requests, call
    // the "close" method on the client to safely clean up any remaining background resources.
```

The samples don't actually show the `close` method on the client being called. Furthermore, the try/with statement in java automatically calls close() and closes the resources, so isn't it confusing to additionally have a comment here telling users to manually call the close method on the client to safely clean up any remaining background resources?

Reply:
The important thing is the comment needs to clarify 2 things:
1. That the client should be reused for multiple requests
2. Call `close()` on the client to clean up background resources.

As you pointed out, they sample _does_ showcase using "close", since it's within a try-with statement. Since users should be re-using the client for multiple requests, it's probably not typical that it'll be convenient to use a try-with statement and will need to call ".close()" manually when they are done with the client.

Reply:
Please add the same comment to your other samples